### ISSUE TITLE
Work around Linux 5.8's unexporting of native_write_cr4

### ISSUE THREAD

**Issue by solardiz** [openwall/lkrg#1]
Linux 5.8 no longer exports the `native_write_cr4` symbol unless `CONFIG_LKDTM` is enabled. We use that symbol, and need to work around that, perhaps by using `kallsyms_lookup_name` to locate it.

Reference: https://www.openwall.com/lists/lkrg-users/2020/08/07/1

**Comment by solardiz** [comment]
We believe this is fixed with 671b079eb985d6b2b24fb943f39cdf48bfce62d6, although more testing is needed. Closing the issue for now (can reopen if necessary).

**Comment by morfikov** [comment]
I can confirm that it builds, at least on:

```
# cat /proc/version
Linux version 5.8.1-amd64 (morfik@morfikownia) (gcc (Debian 10.2.0-5) 10.2.0, GNU ld (GNU Binutils for Debian) 2.35) #1 SMP PREEMPT Thu Aug 13 13:14:40 CEST 2020
```
